app/src/main/java/com/nutomic/syncthingandroid/views/FoldersAdapter.java

```diff
@@ -118,17 +122,15 @@ public View getView(int position, View convertView, @NonNull ViewGroup parent) {
      */
     private static String getLocalizedState(Context c, String state, int percentage) {
         switch (state) {
-            case "idle":     return c.getString(R.string.state_idle);
-            case "scanning": return c.getString(R.string.state_scanning);
-            case "cleaning": return c.getString(R.string.state_cleaning);
-            case "syncing":  return c.getString(R.string.state_syncing, percentage);
-            case "error":    return c.getString(R.string.state_error);
-            case "unknown":  // Fallthrough
-            case "":         return c.getString(R.string.state_unknown);
-        }
-        if (BuildConfig.DEBUG) {
-            throw new AssertionError("Unexpected folder state " + state);
+            case "idle":        return c.getString(R.string.state_idle);
+            case "scanning":    return c.getString(R.string.state_scanning);
+            case "cleaning":    return c.getString(R.string.state_cleaning);
+            case "syncing":     return c.getString(R.string.state_syncing, percentage);
+            case "error":       return c.getString(R.string.state_error);
+            case "unknown":     return c.getString(R.string.state_unknown);
+            case "":            return c.getString(R.string.state_paused);
```

Comment:
You should not do it based on an empty string. You should check that the Folder object has the paused attribute set to true.